### ISSUE TITLE
NullPointerException com.onesignal.UserStatePush.addDependFields

### ISSUE THREAD

**Issue by luck-alex13** [OneSignal/OneSignal-Android-SDK#526]
Hi, I have such crash in my firebase console.
SDK Version is last 3.8.4
OS versions: 93% - Android 4.2.2; 7% - Android 4.1.2

```
Fatal Exception: java.lang.NullPointerException
com.onesignal.UserStatePush.addDependFields (UserStatePush.java:19)
com.onesignal.UserState.generateJsonDiff (UserState.java:100)
com.onesignal.UserStateSynchronizer.internalSyncUserState (UserStateSynchronizer.java:174)
com.onesignal.UserStateSynchronizer.syncUserState (UserStateSynchronizer.java:159)
com.onesignal.UserStateSynchronizer$NetworkHandlerThread$1.run (UserStateSynchronizer.java:69)
android.os.Handler.handleCallback (Handler.java:800)
android.os.Handler.dispatchMessage (Handler.java:100)
android.os.Looper.loop (Looper.java:194)
android.os.HandlerThread.run (HandlerThread.java:60)

**Comment by jkasten2** [comment]
@luck-alex13 This is now fixed in the 3.9.0 OneSignal SDK released today!

**Comment by luck-alex13** [comment]
@jkasten2 Thank you! I will update in the near future
